packages/workbox-precaching/src/index.ts

```diff
@@ -1,22 +1,25 @@
 /*
-  Copyright 2018 Google LLC
+  Copyright 2019 Google LLC
```

Comment:
Nit: I don't think this needs to change for existing files. I'm only mentioned it in the past when a new file was created with a copyright date prior to its existence (due to copy/pasta).